packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.test.tsx

```diff
@@ -125,4 +126,85 @@ describe('DetailsColumn', () => {
 
     expect(mockOnColumnClick.mock.calls.length).toBe(0);
   });
+
+  it('by default, has aria-describedby set for columns which provide an ariaLabel value', () => {
+    const column = assign({}, baseColumn, { ariaLabel: 'Foo' });
+    let component: any;
+    const columns = [column];
+
+    component = mount(
+      <DetailsList
+        items={[]}
+        setKey={'key1'}
+        initialFocusedIndex={0}
+        skipViewportMeasures={true}
+        columns={columns}
+        // tslint:disable-next-line:jsx-no-lambda
+        componentRef={ref => (component = ref)}
+        // tslint:disable-next-line:jsx-no-lambda
+        onShouldVirtualize={() => false}
+      />
+    );
+
+    expect(component.find('[aria-describedby]').length).toBe(1);
```

Comment:
nit: expect(component.find('[aria-describedby]')).toHaveLength(1)

Reply:
TIL, thank you! Fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/7410/commits/f87e493b5bb2f3d908d8904752dc24dbfec5d8e8.